pacai/agents/base.py

```diff
@@ -95,4 +95,4 @@ def _import_agents(glob_path, package_format_string):
             try:
                 __import__(package_format_string % (module_name))
             except ImportError as ex:
-                print("WARN: Unable to import agent: '%s'. -- %s" % (module_name, str(ex)))
+                logging.warning("Unable to import agent: '%s'. -- %s" % (module_name, str(ex)))
```

Comment:
Not going to write this anywhere else, but make sure to use consistent formatting. You switch between double quotes (") and single quotes ('), I think to change the double ones to single.